drools-wb-services/drools-wb-verifier/drools-wb-verifier-guided-decision-table-adapter/src/main/java/org/drools/workbench/services/verifier/plugin/client/builders/ValuesResolver.java

```diff
@@ -377,6 +377,6 @@ private Integer getInteger() {
     private boolean isTypeGuvnorEnum() {
         return conditionColumn != null &&
                 conditionColumn.getValueList() != null &&
-                conditionColumn.getValueList().split(",").length != 0;
+                conditionColumn.getValueList().split(",").length > 1;
```

Comment:
Problem was that "" value list would return true and the Verifier would then think this was an enumeration column.

Reply:
Shouldn't we handle case when single enum is specified?
```java
private boolean isTypeGuvnorEnum() {
        if (conditionColumn != null && conditionColumn.getValueList() != null) {
            if (conditionColumn.getValueList().split(",").length > 1) {
                return true;
            } else {
                return conditionColumn.getValueList().trim().length() > 0;
            }
        } else {
            return false;
        }
    }
```

Reply:
I'll try this out. Might be enough to just check if value list empty without splitting it.

Reply:
I changed it to:
```
    private boolean isTypeGuvnorEnum() {
        return conditionColumn != null &&
                conditionColumn.getValueList() != null &&
                !conditionColumn.getValueList().trim().isEmpty();
    }
```